src/DssDirectDepositHub.sol

```diff
@@ -366,8 +444,13 @@ contract DssDirectDepositHub {
         }
     }
 
-    // --- Allow DAI holders to exit during global settlement ---
-    // wad: should be amount of gems, this could be different than the share tokens
+    /**
+        @notice Allow Users to return vat gem for Pool Shares.
```

Comment:
do you think I should leave it as `return vat gems`? I guess since we removed shares because some pools might not have them, this could just be burning vat.gems for nothing or something more exotic than an ERC-20 share.

Reply:
Actually I don't see an immediate better replacement for the explanations that use `shares` so I guess let's leave as is.